src/mudlet-lua/lua/geyser/GeyserLabel.lua

```diff
@@ -463,6 +463,11 @@ function Geyser.Label:new (cons, container)
   Geyser.Color.applyColors(me)
   me:echo()
 
+  -- Set up mouse hover as the callback if we have one
+  if cons.nestflyout and cons.nestable then
```

Comment:
I have never used geyser label flyouts, so take this with a grain of salt: Is there a particular reason you require both nestable and nestflyout set? Because nestable also sets the click callback on the particular label to be a flyout (which would be already open).

I can imagine people wanting to use the click for a different purpose if they have flyout on hover.

Reply:
nestflyout=true is different from flyOut=true. however if you wanted to reservice the click for something else then you are correct that the entire thing could be pinned on nestflyout=true and simply omit the netable=true parameter. If you think that would be better I can whip that up and test it when I get home.